File: app/schemas/bdo_item.py
from pydantic import BaseModel


class BdoItemBase(BaseModel):
    item_num: int
    item_name_kr: str
    max_enhancement_level: int
    grade: int
    category_num: str
    category_name_kr: str
    tradeable: bool


class BdoItem(BdoItemBase):
    id: int

    class Config:
        orm_mode = True

File: src/main/kotlin/kr/karanda/karandaserver/dto/BDOItem.kt
package kr.karanda.karandaserver.dto

data class BDOItem(
    val itemNum: Int,
    val itemNameKr: String,
    val maxEnhancementLevel: Int,
    val grade: Int,
    val categoryNum: String,
    val categoryNameKr: String,
    val tradeAble: Boolean,
)
